tests/HighLevelTest.hs

```diff
@@ -189,13 +189,24 @@ runPackageTests = do
     do info "Getting package index with etag"
        validateETagHandling "/packages/index.tar.gz"
     do info "Getting testpackage info"
-       xs <- validate NoAuth "/package/testpackage"
+       xs <- validate NoAuth "/package/testpackage.html"
```

Comment:
is adding the html suffix necessary? shouldn't it be html unless otherwise specified?

Reply:
That was part of the original PR. but you're right, it should be html unless otherwise specified. I don't mind leaving it since it's just a call to the endpoint, but  it can be removed if you're more comfortable without it.

Reply:
@gbaz can I get a definitive answer on that?